xAPI.md

```diff
@@ -260,49 +260,30 @@ learning that SCORM could not enable.
 	*	[Appendix G: Cross Domain Request Example](#AppendixG)  
 
 <a name="partone"/>
-#Part One: About The Experience API
+#PART ONE: LEARNING EXPERIENCES
```

Comment:
The caps detnoting the main parts of the document is a convention I've seen used elsewhere to show larger divisions - however I don't feel strongly about it and if you think it detracts from readibility - I will change.

I don't like the original title - as it makes it sound like a sidebar - however I will resubmit with original title and propose a different title in another pull request where I hope it makes sense.